motionplan/ik/combinedInverseKinematics.go

```diff
@@ -13,6 +13,8 @@ import (
 	"go.viam.com/rdk/referenceframe"
 )
 
+const defaultArmJump = 0.00000001
```

Comment:
There shouldn't be any reference to arms in this package.  Lets rename this to defaultJump or something

Reply:
Could also make this public and then use in the constructor that is called in cBiRRT to avoid defining it twice